src/core/core/rpc_engine.cpp

```diff
@@ -850,37 +850,35 @@ namespace dsn {
                             {
                                 resolver->on_access_failure(req2->header->gpid.u.partition_index, err);
 
-                                /*
-                                // TODO(qinzuoyan): because the user will fetch reply states from this response_task 'call',
-                                // so it's useless to create another response_task, which will not be handled by user.
-
                                 // still got time, retry
                                 uint64_t nms = dsn_now_ms();
-                                if (nms + 1000 < timeout_ts_ms)
+                                if (nms + 10 < timeout_ts_ms)
```

Comment:
if we add message_header.client.timeout_ts_ms field, it can be used by both
client side and server side. If you need, then you can set. And the padding
is just a waste.

On Wed, Jul 6, 2016 at 11:14 AM, Zhenyu Guo <notifications@github.com>
wrote:

> In src/core/core/rpc_engine.cpp
> <https://github.com/imzhenyu/rDSN/pull/439#discussion_r69670376>:
>
> >                                  // still got time, retry
> >                                  uint64_t nms = dsn_now_ms();
> > -                                if (nms + 1000 < timeout_ts_ms)
> > +                                if (nms + 10 < timeout_ts_ms)
>
> Good idea. Meanwhile, since it is not necessary on the server side, we can
> put it inside message instead of message_header.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/439/files/78d72cdb0d2846b5fb1d80dea5a4e2e92b985221#r69670376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPVDm01OJhhKCRhwzu_uEgy9KQgfvks5qSx2tgaJpZM4JE9qz>
> .
>